### DISCUSSION TITLE
Entities with "index.ts"

### DISCUSSION THREAD

**Discussion by carloschneider** [mikro-orm/mikro-orm#3700] [Q&A]
Why is it not possible to use entities with "index.ts"?

https://github.com/mikro-orm/mikro-orm/blob/c084dde32860485f6d63872effcaa76b2d35aed1/packages/core/src/metadata/MetadataDiscovery.ts#L156

**Comment by B4nan** [comment]
`index.ts` is a reserved word as it is (almost always) used as a barrel file that only reexports other entities.

**Comment by carloschneider** [comment]
For this case wouldn't it be better to ~have~ use the glob exclude option?

For example:

`entities: ['./src/**/entities/*.js', '!./src/**/entities/index.js']`
